nixos/modules/services/matrix/draupnir.nix

```
@@ -0,0 +1,260 @@
+{ config, lib, pkgs, utils, ... }:
+
+let
+  cfg = config.services.draupnir;
+
+  format = pkgs.formats.yaml {};
+  configFile = format.generate "draupnir.yaml" (pkgs.lib.filterAttrsRecursive (name: value: value != null) cfg.settings);
+in
+{
+  #region Options
+  options.services.draupnir = {
+    enable = lib.mkEnableOption ("Draupnir, a moderation tool for Matrix");
```

```suggestion
    enable = lib.mkEnableOption "Draupnir, a moderation tool for Matrix";
```

Can we say bot here instead of tool?